Makefile

```diff
@@ -92,3 +120,29 @@ bindings:
 		$(CORE_CONTRACTS_ARGS)
 
 	@echo "Bindings generated"
+
+.PHONY: rewardsv2-bindings
+rewardsv2-bindings:
```

Comment:
Why do we need to generate bindings for rewardsv2 and slashing separately? Aren't both part of the same release?

Reply:
due to createQuorum function in registry coordinator. Its not in slashing reg

Reply:
nope afaik

Reply:
Can we use a `sol!` macro instead? I want to avoid splitting the bindings into `slashing` and `m2` over a single contract function.

Reply:
can do . But i don't know if there are other functions also that are missing in slashing that existing consumers may require? 
example : ecdsa bindings are broken rn in slashing  release

Reply:
Right. Can we leave the slashing bindings in `eigen_utils` and have only the M2 ones inside their module?
Example:

```rust
// M2
use eigen_utils::m2_bindings::middleware::operatorstateretriever::OperatorStateRetriever;

// slashing
use eigen_utils::middleware::operatorstateretriever::OperatorStateRetriever;
```

Reply:
sg